City-of-Helsinki/drupal-helfi-sote
# UHF-2508: Hide errand services in sote

How to test:

1. Checkout this branch.
2. `make drush-cr && make drush-cim`
3. If you don't have any services, errand services and service channels on your local yet, import them.
4. Go to view a service that has a lot of errand services and service channels attached such as https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/asukaspysakointi
5. Make sure only the service channels are listed of the errand services and no other information is visible.
